spec/fixtures/purl_fixtures.rb

```diff
@@ -261,21 +224,13 @@ def multi_media_purl
           </resource>
         </contentMetadata>
         <rightsMetadata>
-          <access type="discover">
-            <machine>
-              <world/>
-            </machine>
-          </access>
-          <access type="read">
-            <machine>
-              <world/>
-            </machine>
-          </access>
+          #{access_discover_world}
+          #{access_read_world}
         </rightsMetadata>
       </publicObject>
     XML
   end
-  def multi_resource_multi_media_purl
+  def multi_contentMd_multi_media_purl
```

Comment:
@LynnMcRae says there can only be one contentMetadata block because there can only be one contentMd datastream.  I will change the fixtures accordingly.